src/main/kotlin/no/digdir/accessrequestapi/Application.kt

```diff
@@ -0,0 +1,17 @@
+package no.digdir.accessrequestapi
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration
+import org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan
+import org.springframework.scheduling.annotation.EnableScheduling
+
+@SpringBootApplication(exclude = [MongoAutoConfiguration::class, MongoDataAutoConfiguration::class])
```

Comment:
Trenger ikke eksludere noe du ikke har
```suggestion
@SpringBootApplication
```

Reply:
[Remove MongoAutoConfiguration and MongoDataAutoConfiguration](https://github.com/fellesdatakatalog/access-request-api/pull/3/commits/bd9782e136ce11d31affa9b14b7fac6734b59c48)